lib/src/main/java/graphql/nadel/definition/NadelInstructionDefinition.kt

```diff
@@ -0,0 +1,3 @@
+package graphql.nadel.definition
+
+interface NadelInstructionDefinition
```

Comment:
Renamed from `NadelDefinition`

File too small for Git to pick up on the rename.